flexmeasures/cli/data_add.py

```diff
@@ -1157,3 +1220,52 @@ def parse_source(source):
     else:
         _source = get_or_create_source(source, source_type="CLI script")
     return _source
+
+
+def get_or_create_model(
```

Comment:
This function is potentially useful outside of the CLI context, right? 
Then maybe a better place is somewhere in data/models ...